frontend/src/routes/docs/rooc/+page.svelte

```diff
@@ -21,10 +21,28 @@
         ROOC documentation
     </h1>
     <p>
-        ROOC is a language designed to parse and convert formal optimization models into static formulations. These
-        static formulations can be transformed into linear models which can then be solved using optimization
-        techniques.
+        ROOC is a language designed to help you formalize problems, and prepares those problems in linear form to be solved. The ROOC environment
+        does this in 3 steps:
     </p>
+    <ol>
+        <li>
+        Compile-and-parse step helps formalize your problem into a pre-model (e.g. a LaTeX expression)
```

Comment:
Same thing here as previous comment

Reply:
I removed this as "under the hood" details for now, although maybe we put them back in if we're going to include a brief 3-step example in the "Why Would You Use ROOC?" section